### PR TITLE
put tensorflow template to py

### PR DESCRIPTION
**Pull request by typhoonzero**
Put tensorflow template to py to test.

### CODE REVIEW

File: pkg/sql/codegen/tensorflow/codegen_test.go [hunk 1]
```diff
@@ -59,7 +59,8 @@ func TestTrainCodegen(t *testing.T) {
 				&codegen.NumericColumn{&codegen.FieldMeta{"petal_length", codegen.Float, "", []int{1}, false, nil}},
 				&codegen.NumericColumn{&codegen.FieldMeta{"petal_width", codegen.Float, "", []int{1}, false, nil}}}},
 		Label: &codegen.NumericColumn{&codegen.FieldMeta{"class", codegen.Int, "", []int{1}, false, nil}}}
-	_, err := Train(ir)
+	code, err := Train(ir)
+	fmt.Println(code)
```
Comment (wangkuiyi) [inline pkg/sql/codegen/tensorflow/codegen_test.go]: Should we remove this print?

File: python/sqlflow_submitter/templates/tensorflow/train_template.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        classifier.save_weights(save, save_format="h5")
+        if label_meta["feature_name"] != "":
+            eval_result = classifier.evaluate(validate_input_fn(batch_size), verbose=verbose)
+            print("Training set accuracy: {accuracy:0.5f}".format(**{"accuracy": eval_result[1]}))
+    else:
+        classifier.train(input_fn=lambda:train_input_fn(batch_size))
+        eval_result = classifier.evaluate(input_fn=lambda:validate_input_fn(batch_size))
+        print("Evaluation result:", eval_result)
+
+    print("Done training")
```
Comment (wangkuiyi) [inline python/sqlflow_submitter/templates/tensorflow/train_template.py]: New line at the end of file.

**Review by wangkuiyi — DISMISSED** [review]
LGTM!

File: pkg/sql/codegen/tensorflow/codegen_test.go (hunk 1 above)
Comment (typhoonzero) [reply pkg/sql/codegen/tensorflow/codegen_test.go]: Done.

File: python/sqlflow_submitter/templates/tensorflow/train_template.py (hunk 2 above)
Comment (typhoonzero) [reply python/sqlflow_submitter/templates/tensorflow/train_template.py]: Done.

**Review by wangkuiyi — APPROVED** [review]

**Review by tonyyang-svail — COMMENTED** [review]
@typhoonzero Have we enabled the Python tests?

**Comment by typhoonzero** [comment]
> Have we enabled the Python tests?
@tonyyang-svail Yes, `python -m unittest discover -v python "*_test.py"` in test_units.py will run all python tests.

**Comment by tonyyang-svail** [comment]
Sorry, my bad. I thought the Python files are in `pkg/sql/codegen/tensorflow`.

**Pull request merged**